server/routes/visitors.ts

```diff
@@ -290,11 +295,22 @@ export default function routes(
       res,
     })
 
-    const { offenderNo } = req.session.visitSessionData.prisoner
+    const formValues = (req.flash('formValues')?.[0] as unknown as Record<string, string | string[]>) || {}
```

Comment:
If this is used everywhere, maybe stick in a function?

Reply:
Good suggestion - have done this 👍